plugin-script-malloy/src/main/java/io/kestra/plugin/scripts/malloy/CLI.java

```diff
@@ -0,0 +1,91 @@
+package io.kestra.plugin.scripts.malloy;
+
+import io.kestra.core.models.annotations.Example;
+import io.kestra.core.models.annotations.Plugin;
+import io.kestra.core.models.annotations.PluginProperty;
+import io.kestra.core.runners.RunContext;
+import io.kestra.plugin.scripts.exec.AbstractExecScript;
+import io.kestra.plugin.scripts.exec.scripts.models.DockerOptions;
+import io.kestra.plugin.scripts.exec.scripts.models.ScriptOutput;
+import io.kestra.plugin.scripts.exec.scripts.services.ScriptService;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.*;
+import lombok.experimental.SuperBuilder;
+
+import java.util.List;
+
+@SuperBuilder
+@ToString
+@EqualsAndHashCode
+@Getter
+@NoArgsConstructor
+@Schema(
+    title = "Execute one or more Malloy commands from the Command Line Interface."
+)
+@Plugin(examples = {
+    @Example(
+        full = true,
+        title = "Create a malloy script and run the malloy-cli run command",
+        code = """
+               id: malloy
+               namespace: dev
+               
+               tasks:
+               
+                 - id: working_dir
+                   type: io.kestra.core.tasks.flows.WorkingDirectory
+                   tasks:
+                     - id: local_file
+                       type: io.kestra.core.tasks.storages.LocalFiles
+                       inputs:
+                         model.malloy: |
+                           source: my_model is table('duckdb:https://raw.githubusercontent.com/kestra-io/datasets/main/csv/Iris.csv')
+               
+                           run: my_model -> {
+                               group_by: variety
+                               aggregate:
+                                   avg_petal_width is avg(petal_width)
+                                   avg_petal_length is avg(petal_length)
+                                   avg_sepal_width is avg(sepal_width)
+                                   avg_sepal_length is avg(sepal_length)
+                           }
+               
+                     - id: run_malloy
+                       type: io.kestra.plugin.scripts.malloy.CLI
+                       docker:
+                         image: malloy
+                         pullPolicy: NEVER
+                       commands:
+                         - malloy-cli run model.malloy
+                """
+    )
+})
+public class CLI extends AbstractExecScript {
+    @Schema(
+        title = "Docker options when using the `DOCKER` runner"
+    )
+    @PluginProperty
+    @Builder.Default
+    protected DockerOptions docker = DockerOptions.builder()
+        .image("malloy")
```

Comment:
my only concern is about the Docker image 🤔 

I make this works with the following `Dockerfile` locally (with pullPolicy: NEVER)
```
FROM node:latest

WORKDIR /usr/app

RUN npm install duckdb && \
    npm install -g malloy-cli
   ```
   
   and building with `docker build -t malloy . --platform=linux/amd64`
   
I maybe need some help on this to build a proper Docker image that we can host [here](https://github.com/orgs/kestra-io/packages) (while waiting for a proper image coming from Malloy's team)